quick-start.md

```diff
@@ -2,222 +2,87 @@
 
-After the stack deployment is finished you can verify that the wordpress and mysql services are running:
+After the stack deployment is finished you can verify from Kontena Cloud Dashboard that the lb and web services are running or executing `stack ls` command:
 
 ```
-$ kontena stack ls
-NAME                                                         VERSION    SERVICES   STATE      EXPOSED PORTS
-⊝ wordpress                                                  0.3.0      2          running    *:80->80/tcp
+> stack ls
+NAME            STACK                       SERVICES   STATE     EXPOSED PORTS
+⊛ hello-world   kontena/hello-world:1.0.0   2          running   *:80->80/tcp
 ```
 
-You can use the `kontena service` commands to view the resulting configuration of each deployed stack service:
+To test the installed stack, you must connect to the IP address of the host node publishing the `lb` service on TCP port 80. You can use the public IP address of the host node running the service instance displayed as part of the kontena service show output `service show hello-world/lb`:
 
 ```
-$ kontena service show wordpress/wordpress
-test/wordpress/wordpress:
-  stack: test/wordpress
-  status: running
-  image: wordpress:4.6
-  revision: 2
-  stateful: yes
-  scaling: 1
-  strategy: ha
-  deploy_opts:
-    min_health: 0.8
-  dns: wordpress.wordpress.test.kontena.local
-  secrets:
-    - secret: wordpress-mysql-password
-      name: WORDPRESS_DB_PASSWORD
-      type: env
-  env:
-    - WORDPRESS_DB_HOST=mysql
-    - WORDPRESS_DB_USER=wordpress
-    - WORDPRESS_DB_NAME=wordpress
-  net: bridge
-  ports:
-    - 80:80/tcp
-  instances:
-    wordpress-wordpress-1:
-      rev: 2016-11-28 13:51:02 UTC
-      service_rev: 2
-      node: hidden-moon-99
-      dns: wordpress-1.wordpress.test.kontena.local
-      ip: 10.81.128.115
-      public ip: 192.0.2.1
-      status: running
-      exit code: 0
+> service show hello-world/lb
+   ...
+   hello-world/lb/1:
+      scheduled_to: damp-forest-2
+      deploy_rev: 2017-11-10 08:11:10 UTC
+      rev: 2017-11-10 08:11:10 UTC
+      state: running
+      containers:
+        hello-world.lb-1 (on damp-forest-2):
+          dns: lb-1.hello-world.quick-start.kontena.local
+          ip: 10.81.128.41
+          public ip: 18.194.195.204
+          status: running
 ```
 
-To test the wordpress service, you must connect to the IP address of the host node publishing the wordpress service on TCP port 80.
-You can use the public IP address of the host node running the service instance displayed as part of the `kontena service show` output.
-**Note:** For the special case of using Vagrant for the Kontena setup, you must use the *private* IP address of the node running the `wordpress/wordpress` service: `kontena node show hidden-moon-99 | grep 'private ip'`.
+You can also browse services on Kontena Cloud Dashboard to see stats, logs and configuration of each deployed stack service.
```

Comment:
You can also browse services on the Kontena Cloud Dashboard to see stats, logs and configuration of each deployed stack service.